src/amo/utils/index.js

```diff
@@ -95,7 +95,12 @@ export const checkInternalURL = ({
   const currentHost = url.parse(baseURL).host || '';
   const isForCurrentHost =
     currentHost === urlParts.host || urlString.startsWith(`//${currentHost}`);
-  const isInternal = isRelative || isForCurrentHost;
+
+  // Treat blog URLs as external.
+  const excludePathRegex = /^\/blog\//;
```

Comment:
I feel like external-ish urls should be in a list in config/default.js (like `validLocaleUrlExceptions` which is kinda the same list of AMO urls that aren't served by addons-frontend, but not) but I won't fight for it given it's just one url.

Reply:
I had initially thought about it being a list (and maybe in `config`), but, as you say, it's just one path for now, and might continue to be indefinitely.